sklearn/decomposition/tests/test_incremental_pca.py

```diff
@@ -24,16 +27,43 @@ def test_incremental_pca():
     X_transformed = ipca.fit_transform(X)
 
     np.testing.assert_equal(X_transformed.shape, (X.shape[0], 2))
-    assert_almost_equal(ipca.explained_variance_ratio_.sum(),
-                        pca.explained_variance_ratio_.sum(), 1)
+    np.testing.assert_allclose(ipca.explained_variance_ratio_.sum(),
+                               pca.explained_variance_ratio_.sum(), atol=1e-3)
```

Comment:
unless there's a good reason to do so, we prefer testing based on `rtol` rather than `atol`.